std_navbar-prerelease.ddoc

```diff
@@ -121,6 +121,17 @@ $(DIVID cssmenu, $(UL
 		  $(MODULE3 core, sync, semaphore)
 		)
 	  )
+
+	$(MENU_W_SUBMENU $(TT experimental))
+	  $(ITEMIZE
+		$(PACKAGE $(PACKAGE_NAME logger),
+		  $(MODULE4 std, experimental, logger, package),
+		  $(MODULE4 std, experimental, logger, core),
+		  $(MODULE4 std, experimental, logger, filelogger),
+		  $(MODULE4 std, experimental, logger, multilogger),
+		  $(MODULE4 std, experimental, logger, nulllogger)
+		)
+	  )
```

Comment:
I did it so it is clear that there is something called experimental and that is has a logger module.

having it below exception is sort of wired as exception is stable part of phobos and logger is not.